guides/writing-addons/index.md

```diff
@@ -33,6 +33,17 @@ The default way to make a component is to put the implementation in `addon/`, wh
 
 Fortunately, when we run `ember generate component my-component-name` in an addon project, the CLI takes care of all this re-exporting business. It creates the necessary files and code for us. Addon authors don't usually need to think about the `app` directory or do any work in it.
 
+#### `public/`
+
+You can add static assets in the `public/` directory in your addon, and they will automatically be
+merged into the public directory in the host app under `/your-addon-name/`.
```

Comment:
```suggestion
merged into the public directory in the host app under a folder with the name of your addon.
```

This would be the opposite of my other comment so you the example really pops.